### PR TITLE
Polymorphic relationship to photos

### LINKED ISSUE #1457: Photos for seeds
Useful for seed packets as well as showing what seeds look like.

As part of this, the relationship should probably be made polymorphic so some of the queries get easier, e.g. the crop.photos current is only planting photos, but could be harvests as well.

### PR DESCRIPTION
**Pull request by Br3nda**
fixes #1457

### CODE REVIEW

File: app/models/photographing.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Photographing < ActiveRecord::Base
```
[the remark this answers is not in the document]
Comment (Br3nda) [reply app/models/photographing.rb]: i think this for Rails 5.

**Comment by Br3nda** [comment]
It's letting a photo  e added to an item more than once. I think that's the last thing to fix.

**Comment by Br3nda** [comment]
@maco remember we tried this a year ago and it didn't work? We've shaved enough yaks now. Adding you as a reviewer.

**Comment by Br3nda** [comment]
When we upgrade rubocop, we'll be able to tell it which version of rails we are using.

**Comment by Br3nda** [comment]
Merging #1474 should make this PR green

File: app/views/seeds/_actions.html.haml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  = link_to new_photo_path(id: seed.id, type: 'seed'), class: 'btn btn-default btn-xs' do
+    %span.glyphicon.glyphicon-camera{ title: "Add photo" }
+    Add photo
+- if can? :destroy, @seed
+  = link_to @seed, method: :delete, data: { confirm: 'Are you sure?' }, class: 'btn btn-default btn-xs' do
+    %span.glyphicon.glyphicon-trash{ title: "Delete" }
+    Delete
+
+
+-# - if can?(:edit, planting) || can?(:destroy, planting)
```
Comment (pozorvlak) [inline app/views/seeds/_actions.html.haml]: Please delete commented-out code here.

**Comment by Br3nda** [comment]
i want to add some feature specs for seed photos - will probably add tonight

**Comment by Br3nda** [comment]
I've added pagination to the photos
and i've used faker on the factorybot for photos so the specs can tell the photos apart when checking for them.

**Pull request merged**